rules/windows/persistence_scheduled_task_creation_via_script_interpreter.toml

```diff
@@ -1,28 +1,34 @@
 [metadata]
 creation_date = "2020/02/18"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2021/03/15"
 
 [rule]
 author = ["Elastic"]
 description = "A scheduled task can be used by an adversary to establish persistence, move laterally, and/or escalate privileges."
 false_positives = ["Legitimate scheduled tasks may be created during installation of new software."]
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
-name = "Local Scheduled Task Commands"
+name = "Scheduled Task Creation via Script Interpreter"
 risk_score = 21
 rule_id = "afcce5ad-65de-4ed2-8516-5e093d3ac99a"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:schtasks.exe and
-  process.args:(-change or -create or -run or -s or /S or /change or /create or /run)
+process where event.type == "start" and
```

Comment:
@bm11100 reviewing a sample with @dstepanic17 we would recommend to include also unsigned parents (using a sequence), something like below:

`
sequence with maxspan=5m 
[process where event.type != "end" and 
  (process.name : ("cmd.exe", "wscript.exe", "rundll32.exe", "regsvr32.exe", "wmic.exe", "mshta.exe","powershell.exe", "pwsh.exe", "WmiPrvSe.exe", "wsmprovhost.exe", "winrshost.exe") or process.code_signature.trusted == false)] by process.entity_id
  [process where event.type == "start" and
  (process.name : "schtasks.exe" or process.pe.original_file_name == "schtasks.exe") and process.args : ("/create", "-create") and
  process.args : ("*/RU*", "*/SC*", "*/TN*", "*/TR*", "*/F*", "/XML") and
  /* exclude SYSTEM SID - look for task creations by non-SYSTEM user */
  not user.id : ("S-1-5-18", "S-1-5-19", "S-1-5-20")] by process.parent.entity_id
`

Reply:
updated to sequence, tested against sample and detected